blockly-core/core/ui/fields/field_label.js

```diff
@@ -74,7 +74,9 @@ Blockly.FieldLabel.prototype.init = function(block) {
 
   // Configure the field to be transparent with respect to tooltips.
   this.textElement_.tooltip = this.sourceBlock_;
-  Blockly.Tooltip && Blockly.Tooltip.bindMouseEvents(this.textElement_);
+  if (!this.sourceBlock_.blockSpace.blockSpaceEditor.disableTooltip) {
```

Comment:
Yeah, I originally had a massive `&&` chain, but I'm reasonably convinced all fields will have blocks and all blocks will have all the rest.

Good thought testing that mutator, though! I too often forget about that particular edge case.